apps/aeutils/priv/aeternity_config_schema.json

```diff
@@ -370,6 +370,26 @@
                             "minimum" : 0
                         }
                     }
+                },
+                "protocol_beneficiary_address" : {
+                    "description" :
+                    "Public key of beneficiary account that belongs to protocol maintainers.",
+                    "type" : "string",
+                    "example" : "ak_DummyPubKeyDoNotEverUse999999999999999999999999999",
+                    "pattern": "^ak_[1-9A-HJ-NP-Za-km-z]*$"
+                },
+                "protocol_beneficiary_enabled" : {
+                    "description" :
+                    "If true, the node will split rewards and send part to protocol_beneficiary_address",
+                    "type" : "boolean"
+                },
+                "protocol_beneficiary_factor" : {
+                    "description" :
+                    "Part of rewards to split and send to protocol_beneficiary_address. Rounded to avoid fractions. 100 is 10%",
+                    "type" : "integer",
+                    "default": 100,
```

Comment:
Can code be amended to fetch default from schema i.e. here, and delete `-define(PROTOCOL_BENEFICIARY_FACTOR, 100). %% 10%`? This would enable having only one location for this value, while keeping doc.

Reply:
The same as PROTOCOL_BENEFICIARY_ENABLED (and EXPECTED_BLOCK_MINE_RATE_MINUTES)